src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Authentication.cs

```diff
@@ -44,6 +44,59 @@ private async Task CreateAndValidateRequest(HttpClientHandler handler, Uri url,
 
         public HttpClientHandler_Authentication_Test(ITestOutputHelper output) : base(output) { }
 
+        [Theory]
+        [MemberData(nameof(Authentication_SocketsHttpHandler_TestData))]
+        public async Task SocketsHttpHandler_Authentication_Succeeds(string authenticateHeader, bool result)
```

Comment:
Moved from [`SocketsHttpHandler_HttpClientHandler_Authentication_Test`](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs#L1168-L1204) so that it can be shared between SocketsHttpHandlerTests, SyncHttpHandlerTests and WinHttpHandlerTests (excluded test data failing for it).